tests/test_merge.py

```diff
@@ -43,6 +44,41 @@ def take_first(conflict, _, __, ___):
         self.assertEqual(m.unified_patches,
                          [('change', 'changeme', ('Jo', 'Joe'))])
 
+    def test_continue_run_multiple_conflicts_per_patch(self):
+        lca = {'foo': [{'x': [1, 2, 3]}, {'y': [2, 3, 4]}]}
```

Comment:
Can you try to make simpler test data and remove duplicated code from two tests?

```python
def test_continue_run_multiple_conflicts_per_path(self):
    lca = {'first': [1, 2]}
    first = {'first': [1, 2, 3]}
    second = {'second': 2}
    expected = [
        ('f', {'first': [1, 2, 3], 'second': 2}),
        ('s', {'second': 2}),
    ]

    for resolution, result in expected:
        m = Merger(lca, first, second, {})
       try:
           m.run()
        except UnresolvedConflictsException as e:
           m.continue_run([status for _ in e.content])

        self.assertEqual(patch(m.unified_patches, lca), result)
```